File: Code coach problems/Medium/C++/CMYK_to_RGB.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main()
{
	float c, m, y, k;
	cin >> c >> m >> y >> k;
	
	int r = round(255*(a-c)*(1-k));
	int g = round(255*(1-m)*(1-k));
	int b = round(255*(1-y)*(1-k));
	cout << r << "," << g << "," << b; 
	
	return 0;
}

File: Code coach problems/Medium/Python/CMYK_to_RGB.py
c = float(input())
m = float(input())
y = float(input())
k = float(input())

r = round(255*(1-c)*(1-k))
g = round(255*(1-m)*(1-k))
b = round(255*(1-y)*(1-k))

res = str(r) + "," + str(g) + "," + str(b)

print(res)
